packages/neos-ui/src/localStorageMiddleware.js

```diff
@@ -9,24 +9,31 @@ import {$get} from 'plow-js';
 const localStorageMiddleware = ({getState}) => {
     let timer = null;
     const debounceLocalStorageTimeout = 1000;
-    const persistentActionsPatterns = ['@neos/neos-ui/UI', '@neos/neos-ui/User/Settings'];
+    const persistentActionsPatterns = [
+        '@neos/neos-ui/UI/LeftSideBar/TOGGLE',
```

Comment:
there's a lot of other stuff that should go in: collapsing node creation dialog groups, full screen, probably a lot more